src/Microsoft.FeatureManagement/Targeting/ContextualTargetingFilter.cs

```diff
@@ -21,21 +18,16 @@ public class ContextualTargetingFilter : IContextualFeatureFilter<ITargetingCont
     {
         private const string Alias = "Microsoft.Targeting";
         private readonly TargetingEvaluationOptions _options;
-        private readonly ILogger _logger;
 
         /// <summary>
         /// Creates a targeting contextual feature filter.
         /// </summary>
         /// <param name="options">Options controlling the behavior of the targeting evaluation performed by the filter.</param>
-        /// <param name="loggerFactory">A logger factory for creating loggers.</param>
-        public ContextualTargetingFilter(IOptions<TargetingEvaluationOptions> options, ILoggerFactory loggerFactory)
```

Comment:
I understand we don't use logger currently, but isn't this a breaking change? Could a user derived type use logger? We may need logger in the future.

Reply:
It is a breaking change, changing a public API. If a user derived type uses logger they could always have their own constructor. If we need it in the future shouldn't we add it then?

Reply:
Just want to make sure we are talking about the same thing :). I am seeing you are removing the `ILoggerFactory` parameter from a public API, which is a breaking change. A user code could have called the base constructor and it would be broken. If we do this from the scratch, I would agree with you.

Reply:
I'm not disagreeing it's a breaking change. I'm just saying that after the change, user can still have logging. If we're concerned about a breaking change here we can keep it. But we don't need it, and we are making many breaking changes so it's one of few opportunities to remove it since it's unneeded.

Reply:
If we think chances are we will not add it back in the future, I'm fine to take the opportunity to remove it. I know this is an opportunity. I'm just evaluating the value of the breaking change to the customer.

Reply:
I don't expect us to add it back.

Reply:
All right, sounds fine to remove then.